modules/SofaConstraint/FreeMotionAnimationLoop.cpp

```diff
@@ -303,7 +288,9 @@ void FreeMotionAnimationLoop::step(const sofa::core::ExecParams* params, SReal d
 
 SOFA_DECL_CLASS(FreeMotionAnimationLoop)
 
-int FreeMotionAnimationLoopClass = core::RegisterObject("Constraint solver")
+int FreeMotionAnimationLoopClass = core::RegisterObject(R"(
+The animation loop to use with constraints.
+You must add this loop at the beginning of the scene if you are using constraints.")")
```

Comment:
What is `R` in `core::RegisterObject(R"(` ?

Reply:
https://en.cppreference.com/w/cpp/language/string_literal